docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2023-06-27
```

Comment:
Ahoy, @Sriansh-raj! Thank you for helping to maintain the changelog!

It looks like the badge was unlocked a day early on the 26th of June, would you mind updating the date?
<img width="299" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/7912775e-0d2e-4f33-8f1c-64d44ceea8a5">

Cheers!

Reply:
Well @ParanoidUser idk why is it showing 26 on yours, but in mine it is showing 27th.
<img width="960" alt="image" src="https://github.com/ParanoidUser/yolo/assets/99705533/a97142a8-6993-4253-b8f5-4852bec70a14">

This is the full screenshot of my screen.

Reply:
Makes sense, it's gotta be time zone specific. Thank you for double checking! 👍